packages/backend/src/graphql/resolvers/wallet_address.ts

```diff
@@ -97,8 +98,10 @@ export const createWalletAddress: MutationResolvers<ApolloContext>['createWallet
         addProps.push(toAdd)
       })
 
+    const tenantId = tenantIdToUseAndValidate(ctx, args.input.tenantId)
```

Comment:
@mkurapov 
Unfortunately the `TenantedApolloContext` is an interface. I added this utility function `tenantIdToUseAndValidate` to make the `tenantId` retrieval and validation easier:
```
export function tenantIdToUseAndValidate(
  ctx: TenantedApolloContext,
  tenantId?: string
): string {
  const returnVal = tenantIdToProceed(ctx.isOperator, ctx.tenant.id, tenantId)
  if (!returnVal) {
    const err = WalletAddressError.InvalidTenantIdNotAllowed
    throw new GraphQLError(errorToMessage[err], {
      extensions: {
        code: errorToCode[err]
      }
    })
  }
  return returnVal
}
```